src/zenml/cli/integration.py

```diff
@@ -233,9 +234,9 @@ def export_requirements(
                 "and try again."
             )
         else:
-            formatted_reqs = " ".join([f'"{req}"' for req in requirements])
-            cmd = f"poetry add {formatted_reqs}"
-            os.system(cmd)
+            # Use subprocess.run with shell=False to avoid command injection
+            args = ["poetry", "add"] + requirements
```

Comment:
Shouldn't this still do the escaping?

Reply:
No. Unlike os.system() or a subprocess.run("..", shell=True) (with a string input), we are using a list here with subprocess.run([..]). In such cases, each argument is passed directly without going through a shell. So, we would not have to handle any escaping here.